src/main/java/com/xebialabs/overthere/gcp/GcpSshKey.java

```diff
@@ -0,0 +1,26 @@
+package com.xebialabs.overthere.gcp;
+
+public class GcpSshKey {
```

Comment:
Do we have an intention to allow to extend GcpSshKey? 
If not we need to make class final too.

https://www.oracle.com/java/technologies/javase/seccodeguide.html

Guideline 4-5 / EXTEND-5: Limit the extensibility of classes and methods